cypress/test/AlgoliaSearch.cy.js

```diff
@@ -0,0 +1,26 @@
+import { mount } from 'cypress/react'
+import AlgoliaSearch from '../../components/AlgoliaSearch'
+import { SearchButton } from '../../components/AlgoliaSearch';
+
+describe('AlgoliaSearch component', () => {
+  it('renders without errors', () => {
+    mount(
+      <AlgoliaSearch>
+        <SearchButton>Search</SearchButton>
+      </AlgoliaSearch>
+    )
+  })
+  it('performs search for a specific page', () => {
+    mount(
+      <AlgoliaSearch>
+        <SearchButton>Open Search</SearchButton>
+      </AlgoliaSearch>
+    );
+
+    cy.get('button').click(); // Open the search modal
+
+    cy.get('input[placeholder="Search resources"]').type('welcome')
+    cy.should('have.value', 'welcome')
+  });
```

Comment:
Add the check of getting `Welcome` inside the `docsearch-list` ul list of DOM.

Reply:
Done